lib/route/link.c

```diff
@@ -2273,6 +2284,17 @@ uint8_t rtnl_link_get_carrier(struct rtnl_link *link)
 }
 
 /**
+ * Return carrier on/off changes of link object
+ * @arg link		Link object
+ *
+ * @return Carrier changes.
+ */
+uint32_t rtnl_link_get_carrier_changes(struct rtnl_link *link)
```

Comment:
Most of the other `IFLA_*` "base" link attributes use these simple getters AFAICS, so I thought I stick to the pattern to make usage consistent.

But if you prefer the `rtnl_link_ppp_get_fd` style, I'll update the integer getters accordingly.